# The distance between two decawaves frame = on robot, child_frame = off robot
# Estimated variance ~10 cm
# http://wiki.ros.org/msg
Header header
string child_frame_id
int32 id
float64 distance
int32 distance_quality
int32 quality_factor
float64[] position
